[airparif] NPE at ThingHandlerFactory start

Here's the dump log: 

```
2026-03-09 13:45:14.799 [ERROR] [core.thing.internal.ThingManagerImpl] - Exception occurred while calling thing handler factory 'org.openhab.binding.airparif.internal.AirParifHandlerFactory@19e49918': Cannot invoke "org.openhab.core.thing.binding.ThingHandlerService.setThingHandler(org.openhab.core.thing.binding.ThingHandler)" because "this.serviceInstance" is null
java.lang.NullPointerException: Cannot invoke "org.openhab.core.thing.binding.ThingHandlerService.setThingHandler(org.openhab.core.thing.binding.ThingHandler)" because "this.serviceInstance" is null
	at org.openhab.core.thing.binding.BaseThingHandlerFactory$RegisteredThingHandlerService.initializeService(BaseThingHandlerFactory.java:405)
	at org.openhab.core.thing.binding.BaseThingHandlerFactory.registerThingHandlerService(BaseThingHandlerFactory.java:225)
	at org.openhab.core.thing.binding.BaseThingHandlerFactory.registerServices(BaseThingHandlerFactory.java:160)
	at org.openhab.core.thing.binding.BaseThingHandlerFactory.registerHandler(BaseThingHandlerFactory.java:147)
	at org.openhab.core.thing.internal.ThingManagerImpl.doRegisterHandler(ThingManagerImpl.java:532)
```

--- a/bundles/org.openhab.binding.airparif/src/main/java/org/openhab/binding/airparif/internal/db/DepartmentDbService.java
+++ b/bundles/org.openhab.binding.airparif/src/main/java/org/openhab/binding/airparif/internal/db/DepartmentDbService.java
@@ -59,8 +59,7 @@ public class DepartmentDbService {
         try (InputStream stream = DepartmentDbService.class.getResourceAsStream("/db/departments.json");
                 Reader reader = new InputStreamReader(stream, StandardCharsets.UTF_8)) {
             if (stream == null) {
-                logger.warn("Unable to load departments list: resource /db/departments.json not found");
-                return;
+                throw new IllegalStateException("Resource /db/departments.json not found");
             }
             Gson gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES).create();
             Department[] parsed = gson.fromJson(reader, Department[].class);

